src/com/facebook/buck/features/apple/project/ProjectGenerator.java

```diff
@@ -2362,6 +2361,7 @@ private boolean shouldEmbedSwiftRuntimeInBundleTarget(
                               // a Swift runtime, so it must get bundled to ensure they run.
                               return true;
 
+                            case APPEX:
```

Comment:
This essentially prevent buck from setting ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES to YES. And it does more things other than just not creating the Frameworks directory. I don't feel very confident about this PR. Do we have any other solutions? Thanks
A similar issue: https://github.com/CocoaPods/CocoaPods/issues/4203